group_vars/matrix_servers

```diff
@@ -1204,6 +1214,34 @@ matrix_mautrix_gmessages_database_password: "{{ '%s' | format(matrix_homeserver_
 #
 ######################################################################
 
+######################################################################
+#
+# matrix-bridge-mautrix-wsproxy
+#
+######################################################################
+
+# We don't enable bridges by default.
+matrix_mautrix_wsproxy_enabled: false
+
+matrix_mautrix_wsproxy_systemd_required_services_list: |
+  {{
+    ['docker.service']
+    +
+    (['matrix-synapse.service'] if matrix_synapse_enabled else [])
+    +
+    (['matrix-postgres.service'] if devture_postgres_enabled else []) 
+  }}
+
+matrix_mautrix_wsproxy_homeserver_domain: '{{ matrix_domain }}'
+
+matrix_mautrix_wsproxy_homeserver_address: "{{ 'http://matrix-synapse:8008' if matrix_synapse_enabled else '' }}"
```

Comment:
```suggestion
matrix_mautrix_wsproxy_homeserver_address: "{{ matrix_homeserver_container_url }}"
```

See:


https://github.com/spantaleev/matrix-docker-ansible-deploy/blob/75d4b534844fbcb0df03ad34a2496115786d1529/roles/custom/matrix-base/defaults/main.yml#L143-L146

and

https://github.com/spantaleev/matrix-docker-ansible-deploy/blob/75d4b534844fbcb0df03ad34a2496115786d1529/group_vars/matrix_servers#L436-L443